docs/workforce-management-framework/topics/con-Security-Strategy-for-PassportAuth.adoc

```diff
@@ -0,0 +1,31 @@
+[id='Security-Strategy-Passport.js-{chapter}']
+= Security Strategy for {WFM-RC-NameShort} PassportAuth
+
+== Session Management
+Usage of persistent login sessions using `express-session`.
+Visit link:https://github.com/expressjs/session[express-session] to find more information about the available express
+session options.
+
+== Access Control
+The PassportAuth module allows role based access control. Routes are protected by role as shown in the example.
+
+[source,typescript]
+----
+app.get('/testAdminEndpoint', authService.protect('admin'), (req: express.Request, res: express.Response) => {
+  res.json({ routeName: '/testAdminEndpoint', msg: 'authorized to access admin endpoint' });
+});
+
+----
+
+== Password Storage
+Passport authentication service is using a default custom strategy that is based on Passport.js local strategy.
+Passports local strategy is agnostic about database type. To do this you need to implement the
+link:http://localhost:63342/raincatcher-docs/api/0.0.1/auth-passport/docs/interfaces/_src_user_userrepository_.userrepository.html#getuserbylogin[user repository interface]
+to apply to any data source you require.
+
+== Authentication
+{WFM-RC-NameShort} uses the Default Strategy for username/password authentication. Authentication is done on the cloud
+server. Routes on the cloud server are protected by username, password and role using the protect method.
```

Comment:
"Routes on the cloud server are protected by username, password and role using the protect method."

That doesn't sound right. To access routes user need to login using their credentials. Cookie containing user session key will be used to authenticate and authorize user requesting server endpoint.